src/sentry/integrations/client.py

```diff
@@ -111,13 +112,23 @@ def json(self):
         return self
 
 
-def track_response_code(integration, code):
+def track_response_data(integration, code, error=None, **context):
```

Comment:
@ceorourke I am not sure that you need to specify context like `**context`. I think this would cause `context` to look like this based on how I see it used:
```
{"context": {"org_id": "blah", "integration_id": "blah"}}
```
When we want:
```
 {"org_id": "blah", "integration_id": "blah"}
```
Probably makes more sense to use this as the function definition:
```
def track_response_data(integration, code, error=None, context=None):
```

Reply:
ah okay, I can change that but the last call to `track_response_data` doesn't have access to the error (https://github.com/getsentry/sentry/pull/17289/files/e7db3f0ab2529b7f37b722c445a0b7a8d2c417c0#diff-f1d23e8eced781f96912036609ea3c30R228) so I need to leave it as I can't pass it.

Reply:
Got it, error should be optional